README.md

```diff
@@ -43,28 +43,14 @@ Prerequisites:
 - [Virtual Box](https://www.virtualbox.org/)
 
 
-Then install the Ansible libraries for Postgresql and NodeJS
-
-`ansible-galaxy install geerlingguy.nodejs`
-
-`ansible-galaxy install ANXS.postgresql`
-
-Now you need to setup your virtual machine. 
+Now you need to setup your virtual machine and run the application
```

Comment:
How come you don't need to install these ansible galaxy packages anymore? Or was that never necessary? My understanding was that you have to install them first before you run the Vagrantfile?